README.md

```diff
@@ -23,18 +23,19 @@ Travis
    ```npm``` installed - goto http://nodejs.org/download/ to download installer for your OS.    
    ```ruby``` installed - goto https://www.ruby-lang.org/en/documentation/installation/ to download latest version of ruby.
 
-<br/> Note: On Ubuntu Linux you can use 'sudo apt-get install npm nodejs-legacy' (nodejs-legacy is required to avoid the ""/usr/bin/env: node: No such file or directory" problem).
+<br/> Note: On Ubuntu Linux you can use 'sudo apt-get install npm nodejs-legacy' (nodejs-legacy is required to avoid the "/usr/bin/env: node: No such file or directory" problem).
```

Comment:
```suggestion
Note: On Ubuntu Linux you can use `sudo apt-get install npm nodejs-legacy` (nodejs-legacy is required to avoid the "/usr/bin/env: node: No such file or directory" problem).
```